packages/vscode-extension/src/webview/providers/DependenciesProvider.tsx

```diff
@@ -58,11 +58,11 @@ export default function DependenciesProvider({ children }: PropsWithChildren) {
   }, []);
 
   useEffect(() => {
-    const dependencies: DependencyRecord = {};
+    const newDependencies: DependencyRecord = {};
```

Comment:
While I agree with most "oldSth" and "newSth" renames, here the dependencies is actually the main source of data that is then propagated to state. This object is created once per component lifetime as it is in useEffect with [] passed, and then it is used to accumulate all the updates.

Reply:
you are right, i reverted this change and instead changed the name of constant that was shadowed by this one to `dependenciesDomain`, as it looks like it is an array used only for defining `Dependency` type

dd1023a88cec7a993b4ba0f4ac9e1dd67ede9a5d